process/block/shardblock.go

```diff
@@ -1611,7 +1611,7 @@ func (sp *shardProcessor) createAndProcessMiniBlocksDstMe(
 	}
 	sp.hdrsForCurrBlock.mutHdrsForBlock.Unlock()
 
-	sp.requestMetaHeadersIfNeeded(hdrsAdded, lastMetaHdr)
+	go sp.requestMetaHeadersIfNeeded(hdrsAdded, lastMetaHdr)
```

Comment:
no need for go routine

Reply:
Actually I have added because the call of the new method addHeaderIntoTrackerPool which is inside of underline method. I will remove go from there